revised api docs

edstros opened the pull request:
grammar, tables with code blocks, valid json, the whole enchilada.

edstros committed:
revised api docs

grammar, tables with code blocks, valid json, the whole enchilada.

edstros committed:
Delete CoverCard™-(Also-known-as-MetaMessage).md

edstros committed:
missed a backtick

edstros committed:
fine-tuning code blocks on the tables

edstros committed:
cleanup

made the code blocks pretty, escaped the square brackets in time references so GH didn't think they were links, linked TZ from text rather than have the wikipedia url show.

edstros committed:
Missed a table

Update first table to the pipe format.

edstros committed:
Table cleanup

Missed some code in a table.

edstros committed:
Added space to end of a table

A separate paragraph became associated with a table and needed a line break.

kevinahuber commented:
- [ ] Add table of contents
- [ ] Change developer resource link to `https://help.phyplatform.support/hc/en-us/categories/115000680507-Developer-Resources`
- [ ] Remove link to wiki

edstros committed:
Update readme page

Adds TOC, removes link to wiki, changes dev resources link

edstros committed:
Make subhead smaller, add link to plans page

edstros committed:
Delete Home.md

This content was incorporated as a TOC into README.

edstros committed:
Make TapActions plural

edstros committed:
Update links from wiki to individual pages.

edstros committed:
Update links from wiki to individual pages

edstros committed:
Update links from wiki to individual pages

edstros committed:
Update links from wiki to individual pages

edstros committed:
Update links from wiki to individual pages

edstros committed:
Update links from wiki to individual pages

edstros committed:
Minor change of one Tap Action to Tap Actions

edstros committed:
Update links from wiki to individual pages

edstros committed:
Language changes

[x] Enterprise should read Professional
[x] PHY.net should read Phy Platform
[x] TapAction should read TapActions

edstros committed:
Change PHY.net to Phy Platform
